### PR TITLE
Set up/initialise project and controllers

### PR DESCRIPTION
**Pull request by Beardless-sheik**
In this pull request: 
- Make sure that your project has a Postgres database set up.
- Make sure that your app handles the URLs used in the given wireframes.
- For each URL given, this was created:
  - A route.
  - An action in the correct controller.
  - A view file.
- Regular Ruby on Rails Linter rules are followed

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  get '/', to: "main#index"
+  get '/users', to: "users#index"
+  get '/users/:id', to: "users#show" 
+  get 'users/:userid/posts', to: "posts#index"
+  get 'users/:userid/posts/:id', to: "posts#show"
```
Comment (DeliceLydia) [inline config/routes.rb:9]: - [ ] I suggest that you use `resource` routing allows you to quickly declare all of the common routes for a given resourceful controller you can simply do:
```
   resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Beardless-sheik, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Followed git workflow
- No linter errors

## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: config/routes.rb (hunk 1 above)
Comment (Beardless-sheik) [reply config/routes.rb]: Thanks for the suggestion.....implemented!

**Review by AshakaE — COMMENTED** [review]
Hi @Beardless-sheik 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**